uuid/uuid_test.go

```diff
@@ -88,17 +79,19 @@ func tupleValueIsId(t *testing.T, tuples []interface{}, id uuid.UUID) {
 }
 
 func TestSelect(t *testing.T) {
+	if isUUIDSupported == false {
```

Comment:
`!isUUIDSupported` instead of `isUUIDSupported == false` . Up to you.

Reply:
I prefer more readable version